.github/workflows/test-changed-firestore.yml

```diff
@@ -35,23 +35,27 @@ jobs:
           yarn
       - name: build
         id: build
-        # TODO(wuandy): Separate yarn and egrep into steps, so build failure
-        # is captured by github actions.
-        run: yarn build:changed firestore | egrep "Skipping all"
+        run: |
+          set -o pipefail
+          yarn build:changed firestore | tee ${{ runner.temp }}/yarn.log.txt
```

Comment:
I recommend adding `2>&1` so that stdout and stderr have their lines in the correct relative order.

i.e.

```
yarn build:changed firestore 2>&1 | ...
```

Reply:
Apparently this does not work with `yarn build:changed` for reasons I don't want to find out.